.github/workflows/release.yaml

```diff
@@ -26,4 +26,16 @@ jobs:
         run: sudo snap install charmcraft --classic
       - name: Pack and publish bundle
         run: |
-          make release
+          set -ex
+          # Use a temporary directory to support publishing any of the bundles in this root dir
+          BUILD_DIR=.bundle_build
+          mkdir $BUILD_DIR
+          cp ./releases/latest/mysql-k8s-bundle.yaml $BUILD_DIR/bundle.yaml
+          cp charmcraft.yaml $BUILD_DIR
+          cp README.md $BUILD_DIR
+          cd $BUILD_DIR
+          charmcraft pack --destructive-mode
```

Comment:
This is the method used in other bundles from DPE (and what was in the Makefile).
There's also a `upload-bundle` charming action that would be a more elegant solution and one which I'm not sure that's working.
I'll keep this just because that's not the current PR goal.

Reply:
fwiw, i've created a [task](https://warthogs.atlassian.net/browse/DPE-1476) to standardize these across our bundles